app/properties.json

```diff
@@ -1 +1 @@
            "module_id":"general_params",
            "file_path":"config_properties/general_params.json"
        },
        {
            "description":"Define StP_general_params. you can change the file and save different conbinations.",
            "module_id":"anatomy_params",
            "file_path":"config_properties/process_import_anat.json"
        },
        {
            "description":"Define StP_import_channel. you can change the file and save different conbinations.",
            "module_id":"channel_params",
            "file_path":"config_properties/process_import_channel.json"
        },
        {
            "description":"Define StP_comp_headmodel. you can change the file and save different conbinations.",
            "module_id":"headmodel_params",
            "file_path":"config_properties/process_comp_headmodel.json"
        }
    ]
}
\ No newline at end of file
+{
    "generals":{
        "name":"Brainstorm Protocol for Automatic Head Model",
        "version":"v.81",
        "version_date":"07-16-2023",
        "version_number":81,
        "test_data_url":"",
        "url_check":"https://raw.githubusercontent.com/CCC-members/HCP_BST_source_head_modeler/master/app/app_properties.json",
        "base_url":"https://codeload.github.com/CCC-members/HCP_BST_source_head_modeler/zip/master"
    },
    "matlab_runtime":{
        "mac":"http://ssd.mathworks.com/supportfiles/downloads/R2018b/deployment_files/R2018b/installers/maci64/MCR_R2018b_maci64_installer.dmg.zip",
        "win":"http://ssd.mathworks.com/supportfiles/downloads/R2018b/deployment_files/R2018b/installers/win64/MCR_R2018b_win64_installer.exe",
        "linux":"http://ssd.mathworks.com/supportfiles/downloads/R2018b/deployment_files/R2018b/installers/glnxa64/MCR_R2018b_glnxa64_installer.zip"
    },
    "check_matlab_version":false,
    "check_app_update":false,  
    "run_bash_mode":{
        "description":"HPC-option",
        "value":true
    },    
    "process_files":[
        {
            "description":"Define StP_general_params. you can change the file and save different combinations.",
            "module_id":"general_params",
            "file_path":"config_properties/general_params.json"
        },
        {
            "description":"Define StP_general_params. you can change the file and save different combinations.",
            "module_id":"anatomy_params",
            "file_path":"config_properties/process_import_anat.json"
        },
        {
            "description":"Define StP_import_channel. you can change the file and save different combinations.",
            "module_id":"channel_params",
            "file_path":"config_properties/process_import_channel.json"
        },
        {
            "description":"Define StP_comp_headmodel. you can change the file and save different combinations.",
            "module_id":"headmodel_params",
            "file_path":"config_properties/process_comp_headmodel.json"
        }
    ]
}
```

Comment:
I guess github isn't good for some reason to annotate this (windows end of line double char markers?) one. The diff:

```shell
❯ git diff origin/master... -- app/properties.json
diff --git a/app/properties.json b/app/properties.json
index 5d741d1..0d133b7 100644
--- a/app/properties.json
+++ b/app/properties.json
@@ -1 +1 @@
}   ]   }   "file_path":"config_properties/process_comp_headmodel.json"e file and save different conbinations.",64/MCR_R2018b_glnxa64_installer.zip",
\ No newline at end of file
}   ]   }   "file_path":"config_properties/process_comp_headmodel.json"e file and save different combinations.",64/MCR_R2018b_glnxa64_installer.zip",
\ No newline at end of file
```